packages/icons/src/library/store.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const store = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path
+			fillRule="evenodd"
```

Comment:
I'm assuming both this and the cliprule are needed for this to work? I only see a single Path so it must be a compound shape, I just always get confused when I see these exotic properties.

Reply:
To be honest, so do I. I always remove them and test the output to be sure they are required. In this case, sections of the "awning" in the icon become filled unless the property is included.